src/Parser.svelte

```diff
@@ -41,14 +42,24 @@
           {/each}
         </svelte:component>
       </svelte:component>
-    {:else if type === 'list'}
+    {:else if type === 'list' && ordered}
+      {#if ordered}
       <svelte:component this={renderers.list} {...$$restProps}>
         {#each $$restProps.items as item}
-          <svelte:component this={renderers.listitem} {...item}>
+          <svelte:component this={renderers.orderedlistitem} {...item}>
```

Comment:
What are the advantages of splitting this in different components vs passing the `ordered` prop to the original `listitem` renderer?

I'm mainly asking because right now svelte-markdown's API for the renderers is basically the same as [Marked's](https://marked.js.org), so making this change would make the API differ in this specific case. Both cases would make it differ from Marked's API tho, so I'm weighing pros and cons.

Reply:
My thinking was that this is simpler to style. If I wanted to create a custom marker for unordered lists and one for ordered lists, I would need to put that logic in the list item. With two different renderers, the different renderers don't have any logic needed for styling. I think thats a simpler interface for users of the library.

Reply:
I don't feel super strong on that and could totally be convinced the other way is better also

Reply:
Another way I see it with the current implementation is styling the `List` component instead of `list item` like so:

```css
ul > :global(li) {
  /* Some styles for unordered list items */
}

ol > :global(li) {
  /* Some styles for ordered list items */
}
```

This way it would still be scoped to the parent List without any changes to the current API. But I do see how your proposal might be useful for more fine grained behaviour. What do you think?

Reply:
I like that suggestion. I have implemented it. I think this is cleaner from a user perspective and from an implementation perspective. If I want special styling or behavior, I just need to implement it and it shows up.

Reply:
Perfect! I agree. As a last request, would you mind adding this to the README? 😅 I'll approve it then.

Reply:
Have updated the readme. I also noticed a bug when finishing that up - unordered lists were being skipped. I had left a check in from when I was exploring solutions that didn't need to be there.

Reply:
Nice catch! Didn't notice that at all. Thanks so much for this!

Reply:
Thanks for writing the library. I'd much rather keep this in svelte than render html straight into my page